tests/test_performance.py

```diff
@@ -4,7 +4,7 @@
 import pytest
 
 SAMPLE_AMOUNT = 20
-EXECUTION_TIME_THRESHOLD = 1.0
+EXECUTION_TIME_THRESHOLD = 1.5
```

Comment:
I'm not sure if this is the right way to deal with the problem. IMO it's like ignoring it completely.
I know that retrying failed jobs is also ignoring but at least we know that the test fails randomly.
With higher threshold we won't know if 1s criteria is not met randomly or always.

Reply:
We should try to understand when the problems started. Also do we need to increase by 0.5?

Reply:
We agreed to increase the threshold but at the same time we're starting analysis of decreased performance.